frontend/openchat-client/src/openchat.ts

```diff
@@ -2597,6 +2602,10 @@ export class OpenChat extends EventTarget {
         }
     }
 
+    freezeGroup(chatId: string): Promise<FreezeGroupResponse> {
```

Comment:
As mentioned I think this should probably just return a `Promise<boolean>` and deal with logging the specific error internally to the client library.